sktm/db.py

```diff
@@ -455,23 +456,21 @@ def dump_baseline_tests(self):
                          'baseline.testrun_id = testrun.id')
 
         for (burl, commit, res, buildid) in self.cur.fetchall():
-            print "repo url: %s" % burl
-            print "commit id: %s" % commit
-            print "result: %s" % sktm.tresult(res).name
-            print "build id: #%s" % buildid
-            print "---"
+            print("repo url:", burl)
+            print("commit id:", commit)
+            print("result:", sktm.tresult(res).name)
+            print("build id: #", buildid, sep='')
+            print("---")
 
     def dump_baserepo_info(self):
         self.cur.execute('SELECT url FROM baserepo')
 
         for (burl,) in self.cur.fetchall():
-            print "repo url: %s" % burl
+            print("repo url:", burl)
             stable = self.get_stable(burl)
             latest = self.get_latest(burl)
-            print "most recent stable commit: %s (%s)" % (
-                stable, self.get_commitdate(burl, stable)
-            )
-            print "most recent tested commit: %s (%s)" % (
-                latest, self.get_commitdate(burl, latest)
-            )
-            print "---"
+            print("most recent stable commit: {}({})".format(
+                  stable, self.get_commitdate(burl, stable)))
+            print("most recent stable commit: {}({})".format(
+                  latest, self.get_commitdate(burl, latest)))
```

Comment:
It's awesome how fast you are getting hang of new things! However, can you put the space back in the `{}({})` part in both statements? Format works in a similar way as statements with `%s` do, they don't add any spaces on top of what you specify.

Reply:
Do you mean: `:{}({})`?

Reply:
Do you mean: `: {} ({})`?

Reply:
Okay, I do it now